lib/components/narrative/narrative-itineraries.js

```diff
@@ -148,6 +150,32 @@ class NarrativeItineraries extends Component {
 
     if (!activeSearch) return null
 
+    // Merge duplicate itineraries together and save multiple departure times
+    let nextIndex = 0
+    const mergedItineraries = itineraries.reduce((prev, cur) => {
+      const updatedItineraries = clone(prev)
```

Comment:
What's the reason for cloning prev into a new array? It seems like it's safe to just modify prev.

Reply:
Binh convinced me that this is a case of better safe than sorry